docs/builds.md

```diff
@@ -31,6 +31,45 @@ host OS updates to update the workload containers.  Referring to images
 by tag allows you to use other tooling to dynamically update the workload
 containers.
 
+## Users and groups
+
+This is a highly complex topic.  The short version is that instead of invoking
+e.g. `RUN useradd someuser` in a container build (or indirectly via an RPM
+`%post` script), you should use[sysusers.d](https://www.freedesktop.org/software/systemd/man/latest/sysusers.d.html#).
+
+(Even better, if this is for code executed as part of a systemd unit, investigate
+ using `DynamicUser=yes`)
+
+However, `sysusers.d` only works for "system" users, not human login users.
```

Comment:
is this really true? :thinking: I can set a shell via sysusers.d and have a full capable user (for kiosk for instance, but I think I can login too just fine)

Reply:
Well the man page has

> This tool may be used to allocate system users and groups only, it is not useful for creating non-system (i.e. regular, "human") users and groups, as it accesses
       /etc/passwd and /etc/group directly, bypassing any more complex user databases, for example any database involving NIS or LDAP.

But I think you're right it can be coaxed into doing the allocation part; there was also a discussion on fedora devel about extending this.